src/lib/core/util.c

```diff
@@ -521,8 +521,10 @@ double_compare_nint64(double lhs, int64_t rhs, int k)
 void
 thread_sleep(double sec)
 {
+	assert(sec >= 0 && sec <= TIMEOUT_INFINITY);
+	if (sec < 0 || sec > TIMEOUT_INFINITY)
+		return;
```

Comment:
This is an internal function and we always pass sane values to it. But even if we passed a wrong value, I don't see why that would be a security issue.

AFAIU you're effectively swapping one assertion with an equivalent one. I don't see any point in it.

Please drop this patch and mark the SVACE issue as "False positive".

Reply:
Ok, makes sense to close it.